bots/npm-update

```diff
@@ -75,9 +88,16 @@ def run(package, verbose=False, **kwargs):
         packages = data["dependencies"].keys()
         random.shuffle(packages)
 
-    # Add a tilde to all package.json dependencies
+    def prefix(name, version):
+        if version[0].isalpha():
+            return version
+        if name == package or name not in FRAGILE:
```

Comment:
The first condition appears odd to me - isn't `package == "cockpit"` here? A package should never contain itself as a dependency for sure. Did you perhaps mean something else here?

Reply:
The package is the argument to this bot. In general this bot is invoked as:

     [ ] npm-update

or
     $ bots/npm-update

But if it is invoked as the following, then package is non `None` and its being explicitly invoked by someone (a human) to update a specific dependency. I would like to be less "caution" when explicitly invoked, and use the `^` hat:

    [ ] npm-update angular

or:

    $ bots/npm-update angular

Reply:
Indeed, makes sense. Thanks!